instrumentation/lettuce/lettuce-5.1/src/test/groovy/io/opentelemetry/javaagent/instrumentation/lettuce/v5_1/LettuceReactiveClientTest.groovy

```diff
@@ -301,12 +301,12 @@ class LettuceReactiveClientTest extends AgentTestRunner {
           errored false
           childOf span(0)
           attributes {
-            "${SemanticAttributes.NET_TRANSPORT.key()}" "IP.TCP"
-            "${SemanticAttributes.NET_PEER_IP.key()}" "127.0.0.1"
-            "${SemanticAttributes.NET_PEER_PORT.key()}" port
-            "${SemanticAttributes.DB_CONNECTION_STRING.key()}" "redis://127.0.0.1:$port"
-            "${SemanticAttributes.DB_SYSTEM.key()}" "redis"
-            "${SemanticAttributes.DB_STATEMENT.key()}" "SET key<a> value<1>"
+            "${SemanticAttributes.NET_TRANSPORT.key}" "IP.TCP"
+            "${SemanticAttributes.NET_PEER_IP.key}" "127.0.0.1"
+            "${SemanticAttributes.NET_PEER_PORT.key}" port
+            "${SemanticAttributes.DB_CONNECTION_STRING.key}" "redis://127.0.0.1:$port"
+            "${SemanticAttributes.DB_SYSTEM.key}" "redis"
+            "${SemanticAttributes.DB_STATEMENT.key}" "SET a ?"
```

Comment:
The rendering is a great improvement, thanks! But the spec seems to include argument values for statements

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#redis

I don't know if this is a good idea though - do you mind filing a spec issue to discuss the PII implications and adding scrubbing to the spec?

Reply:
Forgot to mention the improved rendering is very nice!

Reply:
I think this is in accordance with spec - see `db.statement` attribute spec:

```
[3]: The value may be sanitized to exclude sensitive information.
```

Still, I think that this could be described in a more clear and explicit way. I'll create a spec issue to add an example for masking/sanitization.

Reply:
> Forgot to mention the improved rendering is very nice!

Thanks 😄

Reply:
https://github.com/open-telemetry/opentelemetry-specification/issues/1106